### ISSUE TITLE
Do you have tensorflow.keras code for rank distill?

### ISSUE THREAD

**Issue by shijing001** [tensorflow/ranking#303]
Congrats to your AISTATS2021 paper, entitled RankDistill: Knowledge Distillation for Ranking. Could you please share me some piece of code for this paper? Maybe just the coupled distillation loss?

Thank you
